### DISCUSSION TITLE
Using this project for single analog display (Pressure display)

### DISCUSSION THREAD

**Discussion by Mopfus** [jomjol/AI-on-the-edge-device#797] [Q&A]
Hello,

this project seems to be very great and maybe fits for my problem.
I would like to read an analog pressure display with this project to gain digital values. It has only one "pointer" and it looks like this:

![druck](https://user-images.githubusercontent.com/80747454/166718748-c1484123-1caf-4bd0-969b-6574953d3ff5.png)

Is this possible with the project?
- If yes: Do I have to pay attention to something special? 
- If no: Is there another project that anyone can recommend?

(Maybe someone has done something similar.)

Thank you and many greetings
Mopfus

**Comment by jomjol** [comment]
Technically possible, but you need to great and train a dedicated training set.

**Reply by Mopfus** [reply]
Okay. I tohught as the original software can handle pointer gauges it should be no problem. But probably the training set is not for this type of pointer I have.
Is this very complicated to train it by myself? It is no problem to make several Photographs of my analog gauge with different values. Is there any good tutorial or advices how to do this? Or at least a point where I can start?
Thank you in advance.

**Comment by jomjol** [comment]
That is a bit of work. But if you have some python and AI skills, it could be doable: https://github.com/jomjol/neural-network-analog-needle-readout
